nginx/nginx.conf

```diff
@@ -0,0 +1,20 @@
+events {}
+http {
+    server {
+        listen 80;
+        server_name ttl.sh;
+
+        location /v2 {
+            proxy_pass https://registry.ttl.sh;
+            proxy_set_header Host $host;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
```

Comment:
For registry, we probably want buffering turned off
```
    proxy_request_buffering off;
```

Also default max body size is really small with nginx. So something like
```
client_max_body_size 10000m;
```